### PR TITLE
Add collectors to index

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️

### CODE REVIEW

File: src/Storage/FileStorage.php [hunk 1]
```diff
@@ -126,7 +126,10 @@ public function clear(): void
      */
     private function collectIndexData(): array
     {
-        $indexData = ['id' => $this->idGenerator->getId()];
+        $indexData = [
+            'id' => $this->idGenerator->getId(),
+            'collectors' => array_keys($this->collectors),
```
Comment (arogachev) [inline src/Storage/FileStorage.php:131]: Maybe collect keys in the loop below?

**Review by arogachev — APPROVED** [review]

File: src/Storage/FileStorage.php (hunk 1 above)
Comment (xepozz) [reply src/Storage/FileStorage.php:131]: Why?

Comment (arogachev) [reply src/Storage/FileStorage.php:131]: For performance, likely gain is significant though.

Comment (xepozz) [reply src/Storage/FileStorage.php:131]: I don't think there are much difference, but the first option looks better for me. 
Would you like to test? :)

Comment (arogachev) [reply src/Storage/FileStorage.php:131]: It's optional, can be left as is. By the way how big the array size can be?

Comment (xepozz) [reply src/Storage/FileStorage.php:131]: Now it has 11 elements

**Review by samdark — APPROVED** [review]

**Review by rustamwin — APPROVED** [review]

**Pull request merged**